README.md

```diff
@@ -2,6 +2,9 @@
 [![linux](https://github.com/TheAssemblyArmada/Thyme/workflows/linux/badge.svg)](https://github.com/TheAssemblyArmarda/Thyme/actions)
 [![codecov](https://codecov.io/gh/TheAssemblyArmada/Thyme/branch/develop/graph/badge.svg)](https://codecov.io/gh/TheAssemblyArmada/Thyme)
 
+[![en-GB](https://img.shields.io/badge/lang-en--GB-green.svg)](https://github.com/TheAssemblyArmada/Thyme/README.md)
+[![de-DE](https://img.shields.io/badge/lang-de--DE-green.svg)](https://github.com/TheAssemblyArmada/Thyme/README.de-DE.md)
```

Comment:
Do we really need German page for this? Adds maintenance overhead. I would expect that anyone tech savvy will be able to read English, after all source code is English too.

Reply:
Well, was simply an idea as the translation isn't a big deal for me and I can tell you that there're way too much germans not abled to understand english in a acceptable way....

We could integrate the last change date in the top, so one will see if it's outdated